### DISCUSSION TITLE
Add ZeroLogger  => Uptrace Logging

### DISCUSSION THREAD

**Discussion by AchoArnold** [uptrace/opentelemetry-go-extra#79] [Q&A]
Hello,

Is it possible to add an easy hook to push zero logger logs to uptrace? 

https://github.com/rs/zerolog

**Comment by vmihailenco** [comment]
Hi,

No, last time I checked Zerolog did not propagate `context.Context` which is required to pass trace id. You can use Zerolog with Vector though, see https://github.com/uptrace/uptrace-go/tree/master/example/zerolog-vector.

But I would still recommend https://github.com/uptrace/opentelemetry-go-extra/tree/main/otelzap
